### PR TITLE
[CA-1278] Move billing endpoints to v2

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Review by MatthewBemis — APPROVED** [review]
LGTM

**Review by dmohs — APPROVED** [review]

**Review by marctalbott — COMMENTED** [review]
Can we mark this as a draft PR or otherwise note that this should not be merged until PPW is live? The v2 billing endpoints won't create a Google project which will render any Terra billing projects they create unusable until all workspaces have their own Google project

**Comment by s-rubenstein** [comment]
Converted to draft as this should not merge until ppw is released.

File: src/libs/ajax.js [hunk 1]
```diff
@@ -462,7 +462,7 @@ const Billing = signal => ({
 
     return {
       listUsers: async () => {
-        const res = await fetchRawls(`${root}/members`, _.merge(authOpts(), { signal }))
+        const res = await fetchRawls(`${root}`, _.merge(authOpts(), { signal }))
```
Comment (dmohs) [inline src/libs/ajax.js]:
```suggestion
        const res = await fetchRawls(root, _.merge(authOpts(), { signal }))
```

**Review by marctalbott — APPROVED** [review]

**Pull request merged**